packages/components-native/src/Glimmer/Glimmer.tsx

```diff
@@ -83,15 +83,9 @@ export function Glimmer({
         <Svg>
           <Defs>
             <LinearGradient id="gradientShine" x1={0} y1={0.5} x2={1} y2={0.5}>
-              <Stop
-                offset="0%"
-                stopColor={tokens["color-surface--background"]}
-              />
+              <Stop offset="0%" stopColor="rgba(255,255,255,0)" />
```

Comment:
I think we might need to add `stopOpacity={0}`. This is what it looks like if I naively copy the code into `jobber-mobile`...

https://github.com/user-attachments/assets/be7ea3c8-35e3-4a7d-9e7e-4845453f6a32

and after I add the `stopOpacity`

https://github.com/user-attachments/assets/4abbda76-149c-420a-95ca-c0f2bad9bddf

I could also try it with a pre-release build to see if I mucked anything up with my copy pasta.

Reply:
That certainly looks better!

Reply:
@cameron-jobber pushed up the change

Reply:
This is what this looks like now in the pre-release build:

https://github.com/user-attachments/assets/afa554d4-92ac-4fa4-aa2d-c94fea0d94bc

👍 (IMO)

Reply:
This looks good to me @chris-at-jobber ! Thanks for the assist @cameron-jobber

Reply:
FWIW, [here is a PR](https://github.com/GetJobber/jobber-mobile/pull/9581) on `jobber-mobile` that installs the pre-release build. All tests are green. I've just kicked off a Bitrise build if that helps — I'm unsure of what the QA process is for Atlantis updates on JM.

Reply:
@cameron-jobber I can take a quick QA pass on your build and then we'll get this merged!